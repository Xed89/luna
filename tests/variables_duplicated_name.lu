type Program
  fun main()
    let n1 = 57
    let n1 = 14