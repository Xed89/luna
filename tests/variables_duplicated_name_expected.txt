Parsing completed
Syntax Tree:
TypeDeclaration 'Program'
  [0]: 
    Function 'main'
      type:
        null
      statements:
        [0]: 
          DeclarationStatementSyntax
            let n1
              ExpressionLiteralSyntax '57'
        [1]: 
          DeclarationStatementSyntax
            let n1
              ExpressionLiteralSyntax '14'

Compile failed:
Variable with name n1 already declared

